p4runtime_sh/p4runtime.py

```diff
@@ -196,6 +197,7 @@ def handshake(self):
         election_id = arbitration.election_id
         election_id.high = self.election_id[0]
         election_id.low = self.election_id[1]
+        arbitration.role.name = self.role_name
```

Comment:
I do not see why couldn't it be added in a separate PR if needed.

Reply:
That's correct, it can be added in a future PR. However, I wanted to make sure that we are on the same page: without the ability to define the scope of a role, there isn't too much value in supporting role names. You can indeed have multiple roles with different names (and potentially use this information in a P4Runtime server implementation), but in theory all these roles will have the same scope (full pipeline access).